src/esthri/src/lib.rs

```diff
@@ -278,7 +278,7 @@ where
             create_chunk_upload_stream(chunk_stream, s3.clone(), upload_id.clone(), bucket, key)
                 .await;
 
-        let worker_count = Config::global().worker_count();
+        let worker_count = Config::global().worker_count(1.0);
```

Comment:
fractional workers are a thing?

Reply:
Not sure if "workers" is the best term for this, it's basically the number of futures to run in parallel.  The worker thread pool size is ultimately sized by tokio with "magic".

As for the fractional worker/future count, no not really 😅  --  I just needed a way to size counts according to a multiplier.  I was anticipating needing to reduce the count for the next phase of this, but I can forgo this for now and figure out a better abstraction in the next PR.

Reply:
Another possible thing to expose would be the tokio worker thread count (only applies to the cli). right now we use 1 per core